src/verifiers/sev-snp/sevapi.h

```diff
@@ -0,0 +1,72 @@
+/* Copyright (c) 2022 Intel Corporation
+ * Copyright (c) 2020-2022 Alibaba Cloud
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef _SEVAPI_H
+#define _SEVAPI_H
+
+#include <stdint.h>
+
+// Chapter 2 - Summary of Keys
+typedef uint8_t aes_128_key[128 / 8];
+typedef uint8_t hmac_key_128[128 / 8];
+typedef uint8_t hmac_sha_256[256 / 8];
+typedef uint8_t hmac_sha_512[512 / 8];
+
+// Appendix C.1: ALGO Enumeration
+/**
+ * SEV Algorithm cipher codes.
+ */
+typedef enum __attribute__((mode(HI))) SEV_SIG_ALGO {
+	SEV_SIG_ALGO_INVALID = 0x0,
+	SEV_SIG_ALGO_RSA_SHA256 = 0x1,
+	SEV_SIG_ALGO_ECDSA_SHA256 = 0x2,
+	SEV_SIG_ALGO_ECDH_SHA256 = 0x3,
+	SEV_SIG_ALGO_RSA_SHA384 = 0x101,
+	SEV_SIG_ALGO_ECDSA_SHA384 = 0x102,
+	SEV_SIG_ALGO_ECDH_SHA384 = 0x103,
+} SEV_SIG_ALGO;
+
```

Comment:
ALGOs enum is defined in src/include/rats-tls/api.h. Do we need to define the ALOGs enum in sev specify header files?

Reply:
This ALGOs is for sev specifically. ALGOs enum defined in `src/include/rats-tls/api.h` is for rats-tls cert. At present, there are differences in the algorithms, and it is best to distinguish them.

Reply:
OK, thx.